.github/workflows/ci_workflow.yml

```diff
@@ -58,12 +58,13 @@ jobs:
         build: [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12]
         include:
           # -------------------------------------------------------------------
-          # VFX CY2021
+          # GCC, VFX CY2021
           # -------------------------------------------------------------------
           - build: 12
```

Comment:
[Question] Each entry in the matrix must now set several variables, and the number of variables is growing. In order to reduce the number of variables per entry to only the 'changed' ones, is there a way to have some global default values?

Reply:
Good question. I'll have to do some investigation. We're using the GHA matrix in a somewhat unconventional way currently, mostly because the built-in matrix would result in too many permutations.